src/02_spec/07_modules/dem_uart/systemif.rst

```diff
@@ -0,0 +1,63 @@
+System Interface
+----------------
+
+The following signals are required from a bus in order to connect to the DEM.
+
++-----------------+----------+---------------+-------------------------------------------------+
+| Signal          | Width    | Direction     | Description                                     |
++=================+==========+===============+=================================================+
+| ``bus_req``     |    1     | CPU->DEM      | ``1`` indicates an active request from the CPU  |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_addr``    |    3     | CPU->DEM      | Address to be used with write/read operation    |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_write``   |    1     | CPU->DEM      | ``1`` indicates a register write request        |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_wdata``   |    8     | CPU->DEM      | Data to be written into the register            |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_ack``     |    1     | DEM->CPU      | Acknowledge last request                        |
++-----------------+----------+---------------+-------------------------------------------------+
+| ``bus_rdata``   |    8     | DEM->CPU      | Data that was read from the register            |
++-----------------+----------+---------------+-------------------------------------------------+
+
+A new request to this device is made by asserting ``bus_req``, ``bus_write`` indicates whether it
+is a read or write request. A valid register address is presented on ``bus_addr``.
```

Comment:
read (0) or write (1)

"A valid register address is presented on ``bus_addr``." by whom? When?